### PR TITLE
Created transactions list view

### PR DESCRIPTION
**Pull request by ewelina-slepko**

### CODE REVIEW

File: src/app/home/panel/cards/transactions/transactions-list/transactions-list.component.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+.info-date {
+  font-size: .8rem;
+  font-weight: 300;
+}
+
+.info-amount {
+  flex: 1;
+  align-items: flex-end;
+  font-weight: bold;
+}
```
Comment (FilipTalaga) [inline src/app/home/panel/cards/transactions/transactions-list/transactions-list.component.scss:41]: Try to encapsulate styles for naming noise reduction.

File: src/app/home/shared/forms/transaction-form/dtos.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+export interface TransactionDto {
+  name: string;
+  amount: number;
+  date: TimestampDto;
+  budgetId: string;
+  uid: string;
+  id: string;
+}
```
Comment (FilipTalaga) [inline src/app/home/shared/forms/transaction-form/dtos.ts:18]: You can reduce this type, to only contain props that are different than NewTransactionRequest:

```ts
export interface TransactionDto extends Omit<NewTransactionRequest, 'date'>  {
  date: TimestampDto;
  id: string;
}
```

File: src/app/home/panel/cards/transactions/transactions.component.html [hunk 3]
```diff
@@ -1,3 +1,8 @@
-<div @fade>
-<p>transactions works!</p>
+<div class="container" @fade>
+  <h1 class="header">Transactions</h1>
+
+
```
Comment (FilipTalaga) [inline src/app/home/panel/cards/transactions/transactions.component.html:4]: Eyy

File: src/app/home/shared/forms/transaction-form/transaction-form.component.ts [hunk 4]
```diff
@@ -38,15 +43,25 @@ export class TransactionFormComponent implements OnInit {
...
   checkIfStandardCategory(budget) {
     return categories.map(category => category.name).includes(budget.category);
   }
+
+  selectBudget(budget) {
+    this.selectedBudgetId = budget.id;
+  }
+
+  saveTransaction(form: NgForm) {
+    if (form.form.status === 'VALID') {
```
Comment (FilipTalaga) [inline src/app/home/shared/forms/transaction-form/transaction-form.component.ts:59]: Inverse the conditional statement to return the function before any other code. It helps with not creating unnecessary indent.

File: src/app/home/panel/cards/transactions/transactions-list/transactions-list.component.scss (hunk 1 above)
Comment (ewelina-slepko) [reply src/app/home/panel/cards/transactions/transactions-list/transactions-list.component.scss:41]: Ok

File: src/app/home/shared/forms/transaction-form/dtos.ts (hunk 2 above)
Comment (ewelina-slepko) [reply src/app/home/shared/forms/transaction-form/dtos.ts:18]: Ok

File: src/app/home/panel/cards/transactions/transactions.component.html (hunk 3 above)
Comment (ewelina-slepko) [reply src/app/home/panel/cards/transactions/transactions.component.html:4]: :(

File: src/app/home/shared/forms/transaction-form/transaction-form.component.ts (hunk 4 above)
Comment (ewelina-slepko) [reply src/app/home/shared/forms/transaction-form/transaction-form.component.ts:59]: Ok

**Comment by ewelina-slepko** [comment]
Done!

**Pull request merged**